gpkit/nomials/math.py

```diff
@@ -447,74 +406,60 @@ def __init__(self, left, oper, right):
         elif self.oper == ">=":
             m_gt, p_lt = self.left, self.right
         else:
-            raise ValueError("operator %s is not supported by Posynomial"
-                             "Constraint." % self.oper)
+            raise ValueError("operator %s is not supported." % self.oper)
 
         self.p_lt, self.m_gt = p_lt, m_gt
         self.substitutions = dict(p_lt.values)
         self.substitutions.update(m_gt.values)
-        self.unsubbed = self._gen_unsubbed()
+        self.unsubbed = self._gen_unsubbed(p_lt, m_gt)
         self.nomials = [self.left, self.right, self.p_lt, self.m_gt]
         self.nomials.extend(self.unsubbed)
-        self._last_used_substitutions = None
+        self._last_used_substitutions = {}
 
-    def _simplify_posy_ineq(self, exps, cs, pmap):
+    def _simplify_posy_ineq(self, hmap, pmap=None, allow_tautological=True):
         "Simplify a posy <= 1 by moving constants to the right side."
```

Comment:
yeah, fine to reference other docstrings in your docstring, so long as it's not circular :)